### ISSUE TITLE
Create some Sample Data

### ISSUE THREAD

**Issue by caprenter** [ThreeSixtyGiving/sample-data#1]
We talked about creating some story led sample data

**Comment by timgdavies** [comment]
Sounds great. 

Just in case of interest - in the BODS project we've been exploring use of Eduardo and David R's JSON Faker library to create random sample data - which could be useful to help kick-start some other example generation.

**Comment by ekoner** [comment]
@timgdavies agreed. I'll develop some use cases and speak with the dev team about the Faker library.
